src/main/java/slimeknights/tconstruct/library/modifiers/modules/technical/MaxArmorStatModule.java

```diff
@@ -123,33 +114,28 @@ public static void addStat(EquipmentChangeContext context, TinkerDataCapability.
    */
   public static void addStatIfArmor(IToolStackView tool, EquipmentChangeContext context, TinkerDataCapability.TinkerDataKey<Float> key, float amount, boolean allowBroken, @Nullable TagKey<Item> heldTag) {
     if (ArmorLevelModule.validSlot(tool, context.getChangedSlot(), heldTag) && (!tool.isBroken() || allowBroken)) {
-      addStat(context, key, amount);
+      context.getTinkerData().ifPresent(data -> {
```

Comment:
This is still a summation; that is not what we want. A max stat should be something closer to:

```java
float oldLevel = data.get(levelKey, 0f);
float newLevel = MaxArmorLevelModule.updateLevel(data, context, modifierEntry);
data.add(statKey, (newLevel - oldLevel) * scale);
```